src/service/viewport-impl.js

```diff
@@ -1771,6 +1771,12 @@ function getViewportType(win, viewer) {
   if (!isIframed(win) && (getMode(win).localDev || getMode(win).development)) {
     return ViewportType.NATURAL_IOS_EMBED;
   }
+
+  // Enable iOS Embedded mode for iframed tests (e.g. integration tests).
```

Comment:
@aghassemi This is a bit tricky and confusing. We emulate standalone in tests as an iframe. This is because the only way we can run a test is in an iframe. Ideally, we need to be able to indicate whether (while always an iframe) we want to test an iframing viewer case vs standalone case (vs even in-a-box) case. I'm not sure how to communicate all that here... If all tests are ok in this way - we can use this mode and then we can delay accommodating other use cases when we run into them.